backend/src/serverless/dbOperations/handler.ts

```diff
@@ -22,6 +23,8 @@ export async function consumer(event) {
   }
 
   const context = await getUserContext(tenantId)
+  const segmentRepository = new SegmentRepository(context)
+  context.currentSegments = [await segmentRepository.findById(event.segments[0])]
```

Comment:
Shouldn't we use all segments from `event.segments` ?
Where is this `event.segments` coming from? From python worker?

Reply:
From the Git integration. Since it's an integration, it will only ever be 1 segment.

Reply:
Does it make sense to send only one segment in the first place?